html/index.html

```diff
@@ -35,17 +35,17 @@
 
 <span class="sub">bot code:</span>
 
- * <a href="https://github.com/Harrison-M/botsummittalks">@botsummittalks</a> - generate #botsummit talks.
+ * <a href="https://github.com/Harrison-M/botsummittalks">@botsummittalks</a> - Generate #botsummit talks.
  * <a href="https://github.com/jimkang/godtributes">@godtributes</a> - A bot that makes demands for gods appropriately.
  * <a href="https://github.com/jimkang/rapgamemetaphor">@rapgamemetaphor</a> - Posts rap boasts, Everyword-style.
- * <a href="https://github.com/jimkang/autocompleterap">@autocompleterap</a> - Bot that gets raps for pairs of things as suggested by Google autocomplete.
+ * <a href="https://github.com/jimkang/autocompleterap">@autocompleterap</a> - Bot that gets raps for pairs of things suggested by Google autocomplete.
  * <a href="https://github.com/hugovk/schbot">@BotSchmot</a> - A grumpy Twitter bot repeating trending topics with shm-reduplication.
  * <a href="https://github.com/hugovk/everyfinnishno">@EveryFinnishNo</a> - A bot to tweet every Finnish number.
- * <a href="https://github.com/hugovk/word-tools">@favibot, @lovihatibot &amp; @nixibot</a> - A trio of bots searching for things like "I love the word X", saving them to Wordnik lists, tweeting charts, and retweeting some.
- * <a href="https://github.com/hugovk/wotdbot">@AteneumBot, @KiasmaBot &amp; @SinebrychoffBot</a> - A trio of bots to tweet one object from the collection of the Finnish National Gallery: Ateneum, Kiasma and Sinebrychoff art museums.
+ * bots that <a href="https://github.com/hugovk/word-tools">search for tweets about words</a>, create wordnik lists, word clouds, etc.
```

Comment:
Wordnik should have a capital letter.

The bots don't directly create word clouds, although I have manually generated some.